core/src/main/java/de/danoeh/antennapod/core/util/SortOrder.java

```diff
@@ -68,4 +70,12 @@ public static SortOrder fromCodeString(@Nullable String codeStr) {
     public static String toCodeString(@Nullable SortOrder sortOrder) {
         return sortOrder != null ? Integer.toString(sortOrder.code) : null;
     }
+
+    public static SortOrder[] getSortOrderValuesFromStringValues(String[] stringValues) {
```

Comment:
How about calling this `SortOrder.valuesOf`, so it is similar to `SortOrder.valueOf`? The current name contains the name `SortOrder` again, which leads to duplication.

Reply:
Make sense, this name is full of redundancy :) - i just named it first thing that came to mind that will help me guess what method does.